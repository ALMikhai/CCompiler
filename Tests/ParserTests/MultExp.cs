using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace Tests.ParserTests
{
    public partial class ParserTests
    {
        [TestMethod]
        public void MultExp()
        {
            const string fileName = "MultExp";
            Assert.IsTrue(Utils.CheckCorrect(DirectoryPath, fileName, CompileKeys));
        }
    }
}