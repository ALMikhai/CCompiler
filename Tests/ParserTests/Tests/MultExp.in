a * b++ / (int)c % n.m
