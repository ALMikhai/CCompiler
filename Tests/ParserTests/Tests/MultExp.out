\-MULT
  |-a
  \-DIV
    |-b
    | \-INC
    \-MOD
      |-INT
      | \-c
      \-n
        |-DOT
        \-m

